crates/bevy_winit/src/lib.rs

```diff
@@ -146,7 +147,9 @@ impl Plugin for WinitPlugin {
         #[cfg(target_arch = "wasm32")]
         app.add_plugins(CanvasParentResizePlugin);
 
-        let event_loop = event_loop_builder.build();
+        let event_loop = event_loop_builder
+            .build()
+            .expect("Failed creating event loop");
```

Comment:
This should be an `unwrap` with a comment saying this never panics because the system is ran on the main thread.

Reply:
This depends on the target platform. The event loop is not limited to the main thread on them all, see #10020.

Reply:
it can also fail when we attempt to recreate a window after `App:run()` has finished.

Well and definitely other cases listed in https://docs.rs/winit/latest/winit/error/enum.EventLoopError.html

Reply:
I see. Indeed this code can run on any thread. It's just a nit, but I think `.unwrap()` still makes more sense. What information does "Failed creating event loop" adds? Better just `unwrap` and let the error message speak for itself.